build/configs/usbrule.sh

```diff
@@ -17,8 +17,8 @@
 #
 ###########################################################################
 #
-# File   : artik05x_usbrule.sh
-# Description : usb rule script for ARTIK 05X
+# File   : usbrule.sh
+# Description : usb rule script for ARTIK 05X and STM32 Boards
```

Comment:
This is working on imxrt as well. I think we had better remove `for artik05x......`.